test/api/test_gpu.cpp

```diff
@@ -28,6 +28,8 @@
 #include <migraphx/manage_ptr.hpp>
 #include "test.hpp"
 
+// gpu::convolution not supported since MIOpen is OFF
+#if MIGRAPHX_USE_MIOPEN
```

Comment:
Is it necessary to disable the tests? Wont it just fallback to cpu?

Reply:
I think that this should be kept since there is loading conv_relu_maxpool_test.onnx which has pooling layer (gpu::pooling without miopen not merged yet to develop)

Reply:
This makes it harder to figure out how to re-enable the pooling tests as they are all scattered across. It would be better to just have a cpu fallback and just keep all the tests enabled.

Reply:
Do you know how we can make GPU test fallback to CPU. Wouldn't it be misleading for results interpretation since those are GPU tests?

Reply:
We fallback to cpu version for nonmaxsuppression. We do this in lowering so you can follow how it was done for that operator to do something similar for pooling:

https://github.com/ROCm/AMDMIGraphX/blob/c6fc4c18f7ac63f1af5916ad2e8e53c06a656c59/src/targets/gpu/lowering.cpp#L358

Reply:
Pushed changes to fallback pooling, lrn and backwards convolution to CPU and undo changes to skip tests when MIOpen is disabled